build-suggestions/4.8.yaml

```diff
@@ -2,10 +2,10 @@
 # If you specify an arch it must have the full set of values
 ---
 default:
-  minor_min: 4.7.0
+  minor_min: 4.7.11
```

Comment:
I think we should use 4.7.7 as that's the version we had the stable edge from 4.6.

Reply:
updated to set it at 4.7.7